rave/src/main/java/com/uber/rave/BaseValidator.java

```diff
@@ -689,6 +712,28 @@ protected static ValidationContext getValidationContext(Class<?> clazz) {
                         + stringBuilder.toString()));
     }
 
+    private static List<RaveError> createLongDefError(
+            long value,
+            long[] acceptableValues,
+            ValidationContext validationContext) {
+        StringBuilder stringBuilder = new StringBuilder();
+        boolean first = true;
+        for (long string : acceptableValues) {
+            if (!first) {
+                stringBuilder.append(" ");
+            } else {
+                stringBuilder.append("{");
+            }
+            stringBuilder.append(string);
+            first = false;
+        }
+        stringBuilder.append("}");
+
+        return createNewList(
+                new RaveError(validationContext, value + " " + RaveErrorStrings.LONG_DEF_ERROR
```

Comment:
Use `String.format` rather than hard-to-read concatenation

Reply:
+1